### PR TITLE
DISTMYSQL-35 Uninstall instructions

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]

File: source/unistalling.rst [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+   
+   .. code-block:: bash
+   
+      $ sudo systemctl stop mysql 
+       
+#. Remove |PS|.
+   
+   .. code-block:: bash
+    
+      $ sudo apt-get remove percona-server*
```
Comment (ghost) [inline source/unistalling.rst:37]: We need to use purge command instead remove if we want to delete configuration files too.  As example if you have installed percona-server-server packages and was tried to remove it you will have following configuration files:
dpkg -L percona-server-server
/etc
/etc/default
/etc/default/mysql
/etc/init.d
/etc/init.d/mysql
/etc/mysql
/etc/mysql/mysql.cnf
/etc/mysql/mysql.conf.d
/etc/mysql/mysql.conf.d/mysqld.cnf
When are you using purge command for mysql you will ask additional question about need to delete configuration files or not.

**Review by ghost — CHANGES_REQUESTED** [review]

File: source/unistalling.rst (hunk 1 above)
Comment (EvgeniyPatlan) [reply source/unistalling.rst:37]: I disagree. It is up to user if he need to remove configuration or not

Comment (ghost) [reply source/unistalling.rst:37]: @EvgeniyPatlan at least we need to write about it like an option.
If you want to delete configuration file use purge if not use remove. But when you use purge command you will have context menu where you can choose delete configuration files or not. And here user can choose.

Comment (EvgeniyPatlan) [reply source/unistalling.rst:37]: I assume we shouldn't give users additional button to remove everything they need to restore configuration in future..
purge will also remove the datadir!
Users might create automation and they will not see any menu but the cost of error is very high!

Comment (ghost) [reply source/unistalling.rst:37]: Anyway users need to know that remove command delete only packages and not datadir and configuration files.

Comment (nastena1606) [reply source/unistalling.rst:37]: There is a note in Percona Server docs about apt-get remove vs apt-get purge: https://www.percona.com/doc/percona-server/LATEST/installation/apt_repo.html#uninstalling-percona-server
I'm giving the link to this doc in See also section. I don't think it makes sense to repeat the same content twice.

**Review by ghost — APPROVED** [review]

**Pull request merged**